site/_guides/external-authorization.md

```diff
@@ -36,7 +34,7 @@ the request to the upstream application.
 It is helpful to have a mental model of how requests flow through the various
 servers involved in authorizing HTTP requests.
 The flow diagram below shows the actors that participate in the successful
-authorization of a HTTP request.
+authorization of an HTTP request.
```

Comment:
is this a US vs AUS thing? 😂

Reply:
I dunno, I just copy into google docs and let it find the issues. 🤷

Reply:
"a HTTP request" is correct IMHO, but I suppose either could be acceptable

Reply:
heh, "a HTTP request" sounds wrong to my ear, but whatever.